packages/airnode-feed/src/api-requests/data-provider.ts

```diff
@@ -27,6 +27,10 @@ export const callApi = async (
       );
       logger.debug('Performing API call.', { processedEndpointParameters });
 
+      if (!endpoint.operation && isEmpty(endpoint.fixedOperationParameters)) {
+        return;
```

Comment:
Just to note, I'd prefer to mimic exactly same behaviour with airnode which directs output of pre-processing to post-processing. One can see [here](https://github.com/api3dao/airnode/blob/59c8cd8c93f002460649e6c707abe943f0fb2b1d/packages/airnode-node/src/api/index.ts#L315).
```suggestion
        return { data: processedEndpointParameters };
```

Nit: it'd be better to inform about skipping API call in debug mode like following: 
```ts
logger.debug('Skipping API call.', { processedEndpointParameters });
```

Reply:
Agree with Bedirhan here. I'd also add to only print out:
```ts
logger.debug('Performing API call.', { processedEndpointParameters });
```

if we are actually making an API call.

Reply:
I've added logging statements for both skipped and executed API call paths, LMK if this is not what you wanted.

Thanks @bdrhn9 I don't know how I missed that :)